CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## vNext (TBD)
 
 ### Enhancements
-* None
+* Added the method `App.DeleteUserFromServerAsync` to delete a user from the server. The biproduct is that the user is also removed from the device together with their data. (Issue [#2675](https://github.com/realm/realm-dotnet/issues/2675))
```

Comment:
It's "byproduct". 
Otherwise I think we could just say "Additionally the user is also removed...."

Reply:
A general good practice is to avoid passive voice in documentation. This is because we want to convey that an actor does something that leads to a result. With passive voice, things happen, making the source of the occurrence somewhat ambiguous. So in this instance, a more direct statement would be:

```
Added the method `App.DeleteUserFromServerAsync` to delete a user from the server. It will also invalidate the user locally as well as remove all their local data. It will not remove any data the user has uploaded from the server.
```